src/v2/Components/ViewInRoom/index.ts

```diff
@@ -0,0 +1 @@
+export * from "./ViewInRoom"
```

Comment:
@dzucconi - As you're the only one that uses them across Force / Eigen, can you elaborate on your fondness for index files a bit? I know we've discussed this before, but we've been trying to factor this pattern out and were almost successful.

Reply:
Makes it clear what can/should be imported from a subfolder, lets you avoid doing the goofy `import Foo from '../Foo/Foo'`

Reply:
> import Foo from '../Foo/Foo'

This is very direct though, and how we've been doing it. If you're insistent on this pattern all good, but we had a whole discussion before you rejoined about working away from this bad habit (according to Node.js creator). It's just more files in the system.

Reply:
Nah, it's fine with me.